pos_order_return/tests/test_pos_order_return.py

```diff
@@ -65,8 +75,8 @@ def setUpClass(cls):
                 "partner_id": cls.partner.id,
                 "pricelist_id": cls.partner.property_product_pricelist.id,
                 "amount_tax": 0,
-                "amount_total": 2700,
-                "amount_paid": 2700,
+                "amount_total": 1350,
```

Comment:
I'm a little bit confused by this changes in the tests setup. Are they really necessary. I mean, it's hard to tell if the changes break the tests as they were before :thinking:

Reply:
It is true that I should not have touched them, but when doing the new tests I realized that they were not consistent, qty * price_unit with respect to the total of the line (price_subtotal), so I ended up deciding to fix it, do you think we could try it with the previous values in __local__ and if they pass the tests and you see the correct changes leave these new ones?

Reply:
Ok, thanks for the explanations :)